mopidy_mpris/frontend.py

```diff
@@ -20,8 +20,8 @@ def on_start(self):
         try:
             self.mpris = Server(self.config, self.core)
             self.mpris.publish()
-        except Exception as e:
-            logger.warning("MPRIS frontend setup failed (%s)", e)
+        except Exception:
+            logger.exception("MPRIS frontend setup failed")
```

Comment:
I just wanted to note that this will change the output from something short like e.g. "MPRIS frontend setup failed (unable to bind to port)" to a full stack trace.

I don't remember the typical errors here, so I guess this is OK.